website/events/views.py

```diff
@@ -66,6 +66,8 @@ def get_context_data(self, **kwargs):
 
         context["slide_size"] = settings.THUMBNAIL_SIZES["slide"]
 
+        context["albums"] = event.album_set.all()
```

Comment:
This should not be necessary, you can just access `event.album_set` in the html file itself :)